packages/api/src/utils/pin.js

```diff
@@ -44,3 +53,57 @@ export function toPinStatusEnum (trackerStatus) {
   }
   return status
 }
+
+/**
+ * Function that returns list of pins considered ok.
+ *
+ * @param {string} cid cid to be looked for
+ * @param {import('@nftstorage/ipfs-cluster').Cluster} cluster
+ * @param {import('@nftstorage/ipfs-cluster').StatusResponse['peerMap']} [peerMap] Optional list of peers, if not provided the fuctions queries the cluster.
+ * @return {Promise.<import('@web3-storage/db/db-client-types').PinUpsertInput[]>}
+ */
+export async function getOKpins (cid, cluster, peerMap) {
+  if (!peerMap) {
```

Comment:
Is it worth mentioning that, as a next step (a todo?) we should look into using this utils from the car upload as well?